android/titanium/src/java/org/appcelerator/titanium/TiApplication.java

```diff
@@ -302,8 +302,7 @@ protected void loadBuildProperties()
 		this.buildHash = "N/A";
 
 		// Attempt to read the "build.properties" file.
-		final String FILE_NAME = "org/appcelerator/titanium/build.properties";
-		try (InputStream stream = getClass().getClassLoader().getResourceAsStream(FILE_NAME)) {
+		try (InputStream stream = KrollAssetHelper.openAsset("Resources/ti.internal/build.properties")) {
```

Comment:
It turns out `KrollAssetHelper` isn't initialized at this point and does not have a reference to `AssetManager` yet. So, this will fail. (I had to borrow your code with the PR I'm working.)

We should probably just read the file via the `AssetManager` directly like this...
```java
try (InputStream stream = getAssets().open("Resources/ti.internal/build.properties")) {
   // ...
}
```